src/saga/console/index.js

```diff
@@ -103,12 +104,18 @@ export function* getConsoleInUse (action) {
   const consoleUsers = sessionsInternal && sessionsInternal
     .filter(session => session.consoleUser)
     .map(session => session.user)
+  yield put(setVmSessions({ vmId, sessions: sessionsInternal }))
 
   if (consoleUsers.length > 0 && consoleUsers.find(user => user.id === userId) === undefined) {
     yield put(setConsoleInUse({ vmId, consoleInUse: true }))
   } else {
     yield put(setConsoleInUse({ vmId, consoleInUse: false }))
-    yield put(downloadConsole({ vmId, usbFilter, hasGuestAgent }))
+    yield put(downloadConsole({
+      vmId,
+      usbFilter,
+      hasGuestAgent,
+      force: sessionsInternal.find(s => s.user.id === userId) !== undefined,
```

Comment:
Since SPICE and VNC are always gonna have `consoleUser === true`, why not search `consoleUsers`?

And again, `force` is needed here because you want to skip SSO if the current user is already connected to a session right?

Maybe renaming `force` to `skipSSO` would make it a little easier to read.  WDYT?

Reply:
> Since SPICE and VNC are always gonna have consoleUser === true, why not search consoleUsers?

I don't quite understand what do you mean?

> Maybe renaming force to skipSSO would make it a little easier to read. WDYT?

Yes, it will be more clear

Reply:
@sjd78 @bond95 re `consoleUsers` all set or still need to do something?